services/order.py

```
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime.date = None,
+) -> Order | Exception:

```

The return type annotation `-> Order | Exception` is not accurate. The function raises exceptions instead of returning them, so the annotation should be `-> Order` to match the actual behavior and checklist requirements.